engine/engine/content/builtins/fonts/default.font

```diff
@@ -0,0 +1,7 @@
+font: "/builtins/fonts/vera_mo_bd.ttf"
+material: "/builtins/fonts/font.material"
+size: 16
```

Comment:
Should we use the same `size` as the previous `system_font` here to make it easier for people that might be using the previous font to migrate?

Reply:
fixed, thx